src/framework/components/anim/controller.js

```diff
@@ -253,6 +256,14 @@ Object.defineProperties(AnimController.prototype, {
             return playable;
         }
     },
+    playing: {
+        get: function () {
+            return this._playing;
+        },
+        set: function (value) {
```

Comment:
A bit odd to have a setter for a property that sounds like it should be read only.

Reply:
Hmm, yeah, good point. I wouldn't expect that to be settable from the naming...

Reply:
This will be private api only. The component has a public read only 'playable' property.